### DISCUSSION TITLE
Why retry on SMTP failer

### DISCUSSION THREAD

**Discussion by jshapiro2** [postalserver/postal#2784] [Development discussions]
Why does postal retry on messages like the following?  Since it could not resolve to the domain that means there is no MX record.  Why not just make these hard bounce?

{
  "message": {
    "id": 25,
    "token": "zPn31pVoXk1D",
    "direction": "outgoing",
    "message_id": "97e939b5-d789-48f4-b3c4-5f3347071916@rp.domain.com",
    "to": "bounce@bounce.com",
    "from": "my@domain.com",
    "subject": "Confirmation 1",
    "timestamp": 1707224278.5516648,
    "spam_status": "NotChecked",
    "tag": null
  },
  "status": "SoftFail",
  "details": "No SMTP servers were available for bounce.com. Tried bounce.com",
  "output": "Could not resolve bounce.com",
  "sent_with_ssl": null,
  "timestamp": 1707226852.5146668,
  "time": null
}

**Comment by willpower232** [comment]
Postal can't tell the difference between "the records don't exist" and "temporary network error" so there is no way of guaranteeing what the issue is and retrying is the safest option.
